x/relationships/handle_tx.go

```diff
@@ -0,0 +1,111 @@
+package relationships
+
+import (
+	abci "github.com/cometbft/cometbft/abci/types"
+	relationshipstypes "github.com/desmos-labs/desmos/v6/x/relationships/types"
+	juno "github.com/forbole/juno/v5/types"
+
+	"github.com/desmos-labs/athena/utils/transactions"
+
+	"github.com/desmos-labs/athena/types"
+)
+
+func (m *Module) HandleTx(tx *juno.Tx) error {
+	return transactions.ParseTxEvents(tx, map[string]func(tx *juno.Tx, event abci.Event) error{
+		relationshipstypes.EventTypeRelationshipCreated: m.parseCreateRelationshipEvent,
+		relationshipstypes.EventTypeRelationshipDeleted: m.parseDeleteRelationshipEvent,
+		relationshipstypes.EventTypeBlockUser:           m.parseBlockUserEvent,
+		relationshipstypes.EventTypeUnblockUser:         m.parseUnblockUserEvent,
+	})
+}
+
+// --------------------------------------------------------------------------------------------------------------------
+
+// parseCreateRelationshipEvent allows to properly handle a relationship creation event
+func (m *Module) parseCreateRelationshipEvent(tx *juno.Tx, event abci.Event) error {
```

Comment:
It would be better to have name `handle{EventName}event` for all `parse{EventName}Event` like what subspaces does.